### PR TITLE
ci: skip download gotestsum if exist

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Long Dai <long0dai@foxmail.com>

# Description

Skip install when `gotestsum` exists.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: tests/dapr_tests.mk [hunk 1]
```diff
@@ -159,7 +159,7 @@ test-deps:
 	# In golang >=1.16 there is a new way to do this with `go install gotest.tools/gotestsum@latest`
 	# But this doesn't work with <=1.15, so we do it the old way for now 
 	# (see: https://golang.org/ref/mod#go-install)
-	GO111MODULE=off go get gotest.tools/gotestsum
+	command -v gotestsum || GO111MODULE=off go get gotest.tools/gotestsum
```
Comment (halspang) [inline tests/dapr_tests.mk:162]: Is `command` recognized on Windows?

Comment (daixiang0) [reply tests/dapr_tests.mk:162]: I think if you run `make`, it is in `sh` or `bash`, it would have `command`.

Comment (artursouza) [reply tests/dapr_tests.mk:162]: I am running e2e tests and we can see if this will break on Windows.

Comment (daixiang0) [reply tests/dapr_tests.mk:162]: Pass :）

**Review by artursouza — APPROVED** [review]

File: tests/dapr_tests.mk (hunk 1 above)
Comment (daixiang0) [reply tests/dapr_tests.mk:162]: Maybe no need to update branch, there is no conflict or any potential conflict.

**Pull request merged**